logger/log.go

```diff
@@ -25,10 +28,11 @@ func (l *structuredLogger) NewLogEntry(r *http.Request) chimiddleware.LogEntry {
 		"path":        r.URL.Path,
 		"remote_addr": r.RemoteAddr,
 		"referer":     r.Referer(),
+		"timestamp":   time.Now().UTC().Format(time.RFC3339),
 	}
 
-	if reqID := getRequestID(r.Context()); reqID != "" {
-		logFields["request_id"] = reqID
+	if reqID := r.Context().Value("request_id"); reqID != nil {
```

Comment:
Clarification: what's stopping us  from using `getRequestID()` ? 

Would it be to avoid having a circular import?

Reply:
it's a private function so it's only accessible in the `api` package

Reply:
ah ok missed the lower case. Thanks for the clarification!